python/lsst/drp/tasks/update_visit_summary.py

```diff
@@ -706,6 +706,8 @@ def run(
                     psf = None
                 output_record.setPsf(psf)
                 sources = psf_star_catalog[psf_star_catalog["detector"] == detector_id]
+                if len(sources) < 1:
```

Comment:
I would normally do `len(sources) == 0`, but I suppose this would do the same.

Reply:
or
```python
if not sources:
    ...
```